setup.py

```diff
@@ -118,6 +96,5 @@ def finalize_options(self) -> None:
             MYPY_DEPENDENCY,
         ]
     },
-    install_requires=["mypy-extensions>=1.0.0"],
```

Comment:
This is still needed I think?

Reply:
Oh, yes it is - I had it in `pyproject.toml` in `dependencies`